### PR TITLE
Add custom binary and measurement option to iptables plugin (for optional ipv6 collection)

### PR DESCRIPTION
**Pull request by dangeist**
### Required for all PRs:

- [ ] Signed [CLA](https://influxdata.com/community/cla/).
- [ ] Associated README.md updated.
- [ ] Has appropriate unit tests.

### CODE REVIEW

File: etc/telegraf.conf [hunk 1]
```diff
@@ -2004,8 +2004,12 @@
 #   ## Setting 'use_lock' to true runs iptables with the "-w" option.
 #   ## Adjust your sudo settings appropriately if using this option ("iptables -wnvl")
 #   use_lock = false
+#   ## Define an alternate executable, such as "ip6tables". Default is "iptables".
+#   # binary = "ip6tables"
 #   ## defines the table to monitor:
 #   table = "filter"
+#   ## Define this to alter the measurement name. Default is "iptables".
+#   # measurement = ""
```
Comment (danielnelson) [inline etc/telegraf.conf]: We actually have built in support for this type of functionality on every [input plugin][]: `name_override = "foo"`

[input plugin]: https://github.com/influxdata/telegraf/blob/master/docs/CONFIGURATION.md#input-configuration

**Comment by dangeist** [comment]
Will do (likely early next week. Away from console for a few days).On Aug 31, 2018 3:09 PM, Daniel Nelson <notifications@github.com> wrote:@danielnelson requested changes on this pull request.

You can use the make fmt target to run go fmt and fix the test issues, though I usually setup my editor to run it on save.

In etc/telegraf.conf:
>  #   ## defines the table to monitor:
 #   table = "filter"
+#   ## Define this to alter the measurement name. Default is "iptables".
+#   # measurement = ""

We actually have built in support for this type of functionality on every input plugin: name_override = "foo"

**Comment by danielnelson** [comment]
When you update your branch the PR is update automatically.  In the last commit it looks like you accidentally committed a backup of your vendor directory...

**Comment by dangeist** [comment]
Going to start over. This PR is all messed up.

**Pull request closed without merging**